modules/monitoring-about-configuring-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-configuring-core-platform-monitoring_{context}"]
+= About configuring core platform monitoring
+
+Edit the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace to configure the following core platform monitoring components: 
```

Comment:
```suggestion
To configure core platform monitoring components, you edit the following elements of the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace. 
```
This suggestion changes the phrase from an imperative step to an assertion. It distinguishes between the actual components and the YAML elements that control the components. 

This suggestion assumes that the user knows why they would want to do this. If you think it might be useful, you could add a sentence describing the benefits of this task or mentioning a common use case.